src/OutputFormatter/JUnitOutputFormatter.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace SensioLabs\Deptrac\OutputFormatter;
+
+use Fhaculty\Graph\Vertex;
+use SensioLabs\AstRunner\AstMap;
+use SensioLabs\Deptrac\ClassNameLayerResolverInterface;
+use SensioLabs\Deptrac\Dependency\Result;
+use SensioLabs\Deptrac\DependencyContext;
+use SensioLabs\Deptrac\DependencyResult\DependencyInterface;
+use SensioLabs\Deptrac\DependencyResult\InheritDependency;
+use SensioLabs\Deptrac\RulesetEngine\RulesetViolation;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Jan Schädlich <janschaedlich@sensiolabs.de>
+ */
+class JUnitOutputFormatter implements OutputFormatterInterface
+{
+    private static $argument_dump_xml = 'dump-xml';
+
+    public function getName(): string
+    {
+        return 'junit';
+    }
+
+    /**
+     * @return OutputFormatterOption[]
+     */
+    public function configureOptions(): array
+    {
+        return [
+            OutputFormatterOption::newValueOption(static::$argument_dump_xml, 'path to a dumped xml file', ''),
+        ];
+    }
+
+    public function finish(
+        DependencyContext $dependencyContext,
+        OutputInterface $output,
+        OutputFormatterInput $outputFormatterInput
+    ) {
+
+        $xml = $this->createXml($dependencyContext);
+
+        if ($dumpXmlPath = $outputFormatterInput->getOption(static::$argument_dump_xml)) {
+            file_put_contents($dumpXmlPath, $xml);
+            $output->writeln(PHP_EOL . '<info>JUnit Report dumped to '.realpath($dumpXmlPath).'</info>');
+        } else {
+            $output->writeln(PHP_EOL . '<info>JUnit Report dump:</info>');
```

Comment:
We should not print any string/line before, because it would be more difficult to pipe the XML-Result into an other programm.

Reply:
Ok, will change this.